scripts/install_hooks.sh

```diff
@@ -12,7 +12,7 @@ cargo clippy --all-targets || { echo "Error: clippy did not pass - aborting push
 
 cargo +nightly fmt -- --check --config unstable_features=true --config imports_granularity=Crate || { echo "Error: format check failed - aborting push. Please run 'cargo +nightly fmt'." ; exit 1 ; }
```

Comment:
This needs to run in `examples/` as well

Reply:
Good point. I'll open a separate PR for that. (#736)